packages/hint/docs/user-guide/concepts/hints.md

```diff
@@ -26,65 +26,74 @@ Complete the following actions to utilize a hint.
-        ]
-    }
-    ```
+   ```json
+   {
+       "hints": [
+           "html-checker:error"
+       ]
+   }
+   ```
 
 > **NOTE**:  For convenience, the previous code snippet uses the short name to
 > refer to the hint package name.
 
-The following example teaches you on how to use custom hints.  To refer to
-custom hints, you must use the full package name.  To use custom hints,
+The following example teaches you on how to use custom hints. To refer to
+custom hints, you must use the full package name. To use custom hints,
 perform the following steps.
 
-1.  To add the `@myOrg/webhint-hint-clever-custom-audit` and
-    `webhint-hint-another-example1` hints to your `package.json` file, run the
-    following command.
+1. To add the `@myOrg/webhint-hint-clever-custom-audit` and
+   `webhint-hint-another-example1` hints to your `package.json` file, run the
+   following command.
 
-    ```bash
-    npm -i -D @myOrg/webhint-hint-clever-custom-audit webhint-hint-another-example1
-    ```
+   ```bash
+   npm -i -D @myOrg/webhint-hint-clever-custom-audit webhint-hint-another-example1
+   ```
 
-1.  Copy the hints in the following code snippet and add it to your `.hintrc`
-    file.
+1. Copy the hints in the following code snippet and add it to your `.hintrc`
+   file.
 
-    ```json
-    {
-        "hints": [
-            "@myOrg/webhint-hint-clever-custom-audit",
-            "webhint-hint-another-example1"
-        ]
-    }
-    ```
+   ```json
+   {
+       "hints": [
+           "@myOrg/webhint-hint-clever-custom-audit",
+           "webhint-hint-another-example1"
+       ]
+   }
+   ```
 
 ## Hint configuration
 
 When you run `webhint` from the command-line interface, you are always in
-control and you decide which hints are relevant to your use-case.  You also
+control and you decide which hints are relevant to your use-case. You also
 specify what `severity` a hint should have. The following table describes the
 allowed `severity` ratings.
 
 | `Severity` value | Details |
 |:--- |:--- |
-| `off` | The `hint` is not run.  The same as deleting the `hint` from the `.hintrc`. |
-| `error` | If the `hint` finds a major issue that affects one or more targeted browsers. The specified content is broken and you should fix immediately. |
-| `warning` | If the `hint` finds an issue.  The specified content is a problem that you should investigate and fix.  The issue may not cause problems in practice. |
-| `hint` | If the `hint` finds a minor issue, such as something to fix.  The specified content should be tracked and may cause problems in the future. The issue does not cause problems, but may become a `warning` in the future. |
-| `information` | The `hint` provides information.  The specified content is highlighted since it is relevant to you.  The information may help identify parts of a feature or provide instances of a feature for tracking. |
+| `off` | The `hint` is not run. The same as deleting the `hint` from the
```

Comment:
If we are going ahead with the table (I like it), we will need a follow up issue in the website to style them.

Reply:
Cool--That'd be in the webhintio/webhint.io repo, right? If so, I can create an issue to track this.

Reply:
Yep, that's the repo :). Thanks!